src/elm/Pages/Pipeline/Model.elm

```diff
@@ -42,23 +48,35 @@ type alias PartialModel a =
         , time : Posix
         , repo : RepoModel
         , shift : Bool
-        , templates : ( WebData Templates, Error )
-        , pipeline : Pipeline
+        , templates : PipelineTemplates
+        , pipeline : PipelineModel
         , page : Page
         , toasties : Stack Alert
+        , sourceRepos : WebData SourceRepositories
+        , user : WebData CurrentUser
     }
 
 
+type alias Msgs msg =
```

Comment:
sad to see this code getting reverted, but in the grand scheme this is a better solution for having shared Msgs across multiple pages